### PR TITLE
Update Python versions and add Python 3.7

### PR DESCRIPTION
**Pull request by hesstobi**
reslove #4

### CODE REVIEW

File: python-versions.txt [hunk 1]
```diff
@@ -1,8 +1,9 @@
...
+3.7.0
+3.6.6
+3.5.6
+3.4.9
 3.3.7
-2.7.14
+2.7.15
 2.6.9
 pypy-5.7.1
 pypy3.5-5.10.1
```
Comment (bittner) [inline python-versions.txt]: This now needs to be updated to `pypy3.5-6.0.0`.

And for some reason there is a `pypy3.3-5.5-alpha` also in [python-versions.txt](https://github.com/themattrix/docker-tox-base/blob/master/python-versions.txt#L9) (but not showing up here). Not sure if that's fine to be omitted (and why it's not showing up in the merged result).

**Comment by bittner** [comment]
@themattrix Any chance this gets merged?

File: python-versions.txt (hunk 1 above)
Comment (hesstobi) [reply python-versions.txt]: I updated `pypy3.5-6.0.0`
`pypy3.3-5.5-alpha` is still in the [python-versions.txt](https://github.com/hesstobi/docker-tox-base/blob/update-python-versions/python-versions.txt). Thats way its not shoing here

**Comment by bittner** [comment]
This now fixes both issue #2 and #4, if I'm not mistaken.

**Comment by bittner** [comment]
Let's keep our fingers crossed that @themattrix gets notified about this (and merges the two PRs). :crossed_fingers: 

:bell: ***ping***

**Pull request merged**